### DISCUSSION TITLE
Is there a way to instantiate None object?

### DISCUSSION THREAD

**Discussion by hovnatan** [facebookresearch/hydra#2856] [Q&A] [answered]
I need to instantiate `None` object. I tried `_target_ : <empty>` that didn't work. Is  there a way to do that?

**Comment by odelalleau** [comment]
Did you try using as `_target_` a callable that returns `None`?

**Reply by hovnatan** [reply]
I created a function returning `None` and that worked. Thanks @odelalleau ! Is there something in Python standard library returning `None` so that I don't write my own function?

**Reply by odelalleau** [reply]
I guess `print()` does :)

**Reply by hovnatan** [reply]
` _target_: builtins.print` worked. Thanks @odelalleau !

**Comment by Jasha10** [comment, marked as answer]
You could also instantiate `types.NoneType` (in python3.10+)
```
from types import NoneType
assert NoneType() is None
```
so `_target_: types.NoneType` could work.
